apps/verification/src/verification.service.ts

```diff
@@ -489,11 +520,11 @@ export class VerificationService {
    * @param payload 
    * @returns Get requested proof presentation details
    */
-  async _sendOutOfBandProofRequest(payload: IProofRequestPayload): Promise<{
+  async _sendOutOfBandProofRequest(payload: IProofRequestPayload | IPresentationExchangeProofRequestPayload): Promise<{
```

Comment:
- In the subsequent PR  change `IProofRequestPayload` to `IndyProofRequestPayload` or something similar to separate it out from presentation exchange payload 
- @ankita-p17  Thoughts ?